proxy/Cargo.toml

```diff
@@ -16,7 +16,7 @@ futures-mpsc-lossy              = { path = "./futures-mpsc-lossy" }
 conduit-proxy-router            = { path = "./router" }
 
 bytes = "0.4"
-domain = "0.2.2"
+domain = "0.2.3"
```

Comment:
to be clear: `0.2.3` is equiv to `^0.2.3`, right?

Reply:
> to be clear: 0.2.3 is equiv to ^0.2.3, right?

Yes, `^` is the default.